text/processing.py

```diff
@@ -112,9 +112,11 @@ class ConcatenateJob(Job):
     Concatenate all given input files (gz or raw)
     """
 
-    def __init__(self, text_files):
+    def __init__(self, text_files, zip_out=True, out_name="out"):
```

Comment:
Do we need an hash exclude for the new parameters or is this job sufficiently unused?

Reply:
Oh missed this, yes for sure this is required!

Reply:
Why do we have no has exclude for the "out_zip" in PipelineJob `https://github.com/rwth-i6/i6_core/blob/main/text/processing.py#L103` ? The output format can change the hash?

Reply:
By hash exclude we do not mean to exclude `out_zip` from the hash computation, but to exclude the old default. Example:
```python3
j = SomeJob(old_feature=xyz)
```
And now you have
```python3
j = SomeJob(old_feature=xyz, new_feature=False)
```
You want the same hash for both.

This is why you need to include:
```python3
__sis_hash_exclude = {'new_feature': False}
```

And yes, the output format should definitely change the hash, because you are getting a different file.

Reply:
got it! thanks